NitroxClient/Communication/Packets/Processors/CyclopsChangeColorProcessor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+using NitroxModel.Helper.GameLogic;
+using NitroxModel.Helper.Unity;
+using NitroxModel.Packets;
+using System;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+	public class CyclopsChangeColorProcessor : ClientPacketProcessor<CyclopsChangeColor>
```

Comment:
We use spaces to indent, not tabs.

You can fix this permanently by setting Visual Studio to use spaces instead of tabs.